Decred-Never-Stops.md

```diff
@@ -0,0 +1,100 @@
+
+At its core, Decred was envisioned not just as another digital currency, but as a new project/workplace for the future of decentralized finance. Spearheaded by visionary minds like tacotime, _ingsoc, and Jake Yocom-Piatt, Decred took shape as a hybrid of Proof-of-Work (PoW) and Proof-of-Stake (PoS) consensus systems, drawing inspiration from the Memcoin2 cryptocurrency.
+
+Decred's journey isn't just about creating a new form of currency; it is about building an ecosystem where innovation thrives. Decred's hybrid model not only ensures security and decentralization but also fosters a culture of continuous development and adaptation. Let's check what makes Decred a unique project!
+
+A Legacy of Innovation
+
+The innovation lies at the core of the Decred project, driving its pursuit of progress in the blockchain ecosystem. Formed by a team of exceptional developers, where many have worked for big companies and coded for big projects, such as Bitcoin, Decred stands out as a fertile ground for forward-thinking solutions and blockchain applications. Decred's unique hybrid consensus model is central to its ability to stay at the forefront of innovation which enables swift adoption of new technologies and methodologies.
+
+Decred's network operates on a unique hybrid consensus protocol, blending two key approaches: Proof of Work (PoW) and Proof of Stake (PoS). In this setup, PoW miners handle the validation of transactions, while PoS stakeholders take charge of voting on network upgrades. This innovative model ensures that PoW miners and PoS stakeholders are rewarded, with block rewards being divided between them.
+
+But innovation requires more than just talent and a hybrid consensus; it requires resources and autonomy. That's where Decred's treasury comes into play. Unlike many other projects that rely on external investors, Decred has its treasury own fund, meticulously tracked and transparently accessible on the blockchain. This fund serves as a pillar of autonomy, ensuring that Decred remains self-sufficient in its pursuit of innovation. 
+
+Anyone can submit a proposal to use the treasury funds on the Politeia platform, Decred's governance platform. Stakeholders have the authority to decide if the funds should be used for a proposal. Politeia creates an environment for discussion and voting, where the community can debate the proposals and give their takes on why money should/shouldn't be used to fund an initiative.
+
+As an example, we have this website: Cypherpunktimes.com. First created as Decred Magazine, this website is a platform to spread information/opinions about the crypto market, the latest blockchain technology, and related news. Stakeholders have found this an important initiative to the marketing of the project, and voted to fund it!
+
+There are many other examples of ongoing development and marketing projects. Right now several are being voted, check out them here ([Politeia link](https://proposals.decred.org/). 
```

Comment:
As this will likely be read in the future I'd change "Right now several are being voted, check out them here" to "which can be viewed on the decred governance platform -link-"